fem/bilininteg.cpp

```diff
@@ -2947,13 +2947,13 @@ void DGTraceIntegrator::AssembleFaceMatrix(const FiniteElement &el1,
                                            FaceElementTransformations &Trans,
                                            DenseMatrix &elmat)
 {
-   int dim, ndof1, ndof2;
+   int dim1, ndof1, ndof2;
```

Comment:
We can probably just use `dim` here like in the other integrators instead of `dim1`.

Reply:
Fixed